utilities/service-utils.js

```diff
@@ -22,18 +22,21 @@ ServiceUtils.prototype = {
   // name may fail because of duplicate usage. This function verifies
   // that the url associated with the service, contains the matched
   // input value, hence reducing the chances of a false match.
-  checkCFForService: function(serviceName, returnBoolean, alchemyRegex) {
-    var regex = alchemyRegex ?
-                RegExp('(http|https)(://)('+serviceName+').*') :
-                RegExp('(http|https)(://)([^\/]+)(/)('+serviceName+').*');
+  checkCFForService: function(serviceName, returnBoolean) {
+    var regex = RegExp('(http|https)(:\/\/)([^\/]+).*('+serviceName+').*');
 
     var services = appEnv.getServices();
 
     for (var service in services) {
-      if (services[service].hasOwnProperty('credentials')) {
-        if (services[service].credentials.hasOwnProperty('url')){
-          if (services[service].credentials.url.search(regex) === 0){
-            return returnBoolean ? true : services[service].credentials;
+      let section = services[service];
+      if (Array.isArray(section)) {
+        section = section[0];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 16 spaces but found 8.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4764899)